### PR TITLE
Include the CTL environment name in the metric names

### PR DESCRIPTION
**Pull request by amarathavale**
This change will allow us to use the same container in different CTL environments: Daily and Staging, and yet be able to share the same Graphite endpoint for emitting metrics. This would ensure metrics from 2 sources are not shared and hence, in the event of any latency issues, we'd know where it is originating from.

Also included a mechanism to allow the container to receive the manageDatabase option through the container parameters. Managing the database i.e. creating it with the desired throughput, can be done by passing this flag.

Tested this out locally + w/ the container. The output appears like the one below when someone explicitly sets ctl_environment to 'Staging' ()the default value is 'Daily')
-- Meters ----------------------------------------------------------------------
Staging GET ctlWorkloadInvitations Document NotFound Operations
...
Staging GET ctlWorkloadInvitations Successful Operations
...
Staging GET ctlWorkloadInvitations Unsuccessful Operations
...
Staging SQL_QUERY ctlWorkloadInvitations Document NotFound Operations
             count = 0
...
Staging SQL_QUERY ctlWorkloadInvitations Successful Operations
...
-- Timers ----------------------------------------------------------------------
Staging GET ctlWorkloadInvitations Latency

### CODE REVIEW

File: sdk/cosmos/azure-cosmos-benchmark/ctl/linkedin/run_benchmark.sh [hunk 1]
```diff
@@ -113,9 +130,19 @@ jvm_opt=""
...
+    fi
 else
-java -Xmx$heap_size -Xms$heap_size  $jvm_opt -Dcosmos.directModeProtocol=$protocol -Dazure.cosmos.directModeProtocol=$protocol -jar "$jar_file" -serviceEndpoint "$service_endpoint" -masterKey "$master_key" -databaseId "$database_name"  -collectionId "$collection_name" -numberOfCollectionForCtl "$number_Of_collection" -throughput $throughput  -consistencyLevel $consistency_level -concurrency $concurrency -numberOfOperations $number_of_operations -operation $operation -connectionMode $connection_mode -maxRunningTimeDuration $max_running_time_duration -graphiteEndpoint $ctl_graphite_endpoint -numberOfPreCreatedDocuments $number_of_precreated_documents -bulkloadBatchSize $bulk_load_batch_size -printingInterval $printing_interval 2>&1 | tee -a "$log_filename"
+    if [ -z "$ctl_manage_database" ]
+    then
```
Comment (simplynaveen20) [inline sdk/cosmos/azure-cosmos-benchmark/ctl/linkedin/run_benchmark.sh:141]: ctl_graphite_endpoint is missing in this condition ?

Comment (amarathavale) [reply sdk/cosmos/azure-cosmos-benchmark/ctl/linkedin/run_benchmark.sh:141]: Good catch. Thanks for the close/careful review.

**Review by simplynaveen20 — APPROVED** [review]
Thanks Amar

**Review by moderakh — APPROVED** [review]
LGTM. wait for Naveen sign off.

**Pull request merged**